doc/hardware/peripherals/sensor/read_and_decode.rst

```diff
@@ -0,0 +1,99 @@
+.. _sensor-read-and-decode:
+
+Read and Decode
+###############
+
+The quickly stabilizing experimental APIs for reading sensor data are:
+
+* :c:func:`sensor_read`
+* :c:func:`sensor_read_async_mempool`
+* :c:func:`sensor_get_decoder`
+* :c:func:`sensor_decode`
+
+
+Benefits over :ref:`sensor-fetch-and-get`
+*********************************************************
+
+These APIs allow for a wider usage of sensors, sensor types, and data flows with
+sensors. These are the future looking APIs in Zephyr and solve many issues
+that have been run into with :ref:`sensor-fetch-and-get`.
+
+:c:func:`sensor_read` and similar functions acquire sensor encoded data into
+a provided buffer by the caller. Decode (:c:func:`sensor_decode`) then
+decodes the sensor specific encoded data into fixed point :c:type:`q31_t` values
+as vectors per channel. This allows further processing using fixed point DSP
+functions that work on vectors of data to be done (e.g. low pass filters, fft,
+fusion, etc).
+
+Reading is by default asynchronous in its implementation and takes advantage of
+:ref:`rtio` to enable chaining asynchronous requests, or starting requests
+against many sensors simultaneously from a single call context.
+
+This enables incredibly useful code flows when working with sensors such as...
+
+* Starting a read for sensors directly from an interrupt handler (no dedicated
+  thread needed!)
+* Starting a read for multiple sensors simultaneously
+* Requesting multiple reads to the same device for Ping-Pong (double buffering)
+  setups.
+* Creating entire pipelines of data flow from sensors allowing for software
+  defined virtual sensors (:ref:`sensing`) all from a single thread with DAG
+  process ordering.
+* Potentially pre-programming DMAs to trigger on GPIO events, leaving the CPU
+  entirely out of the loop in handling sensor events like FIFO watermarks.
+* Obtaining the raw sensor data, decoding never, later, or on a separate
+  processor (e.g. a phone).
```

Comment:
I would suggest, if you haven't already, double-checking this listed is in order of "importance". "Obtaining the raw sensor data [...]" feels to me like it might be more important than some of the other items listed here but I might be wrong and/or everything might be more or less of equal importance